### DISCUSSION TITLE
Functions reference

### DISCUSSION THREAD

**Discussion by manake** [josdejong/mathjs#2780] [Q&A]
Functions reference is linked in many places but doesn't exist: https://github.com/josdejong/mathjs/blob/develop/docs/reference/functions.md

This is probably the highest priority on the documentation page.

Can the function reference be right on the documentation home page like on https://underscorejs.org/? Would be much better IMO.

**Comment by josdejong** [comment]
The documentation can be found here: https://mathjs.org/docs/index.html

Much of the documentation is generated from the source code, and is only available on the website (and not committed in the repo itself).

**Comment by manake** [comment]
Ahh, that's great.

This solves 90% of this question then.

But if the website could be like https://underscorejs.org/ that'd be better than current.

**Reply by josdejong** [reply]
Yeah, so make the function reference more prominent. That's a good idea. I don't like the current Google search that much, it's having quite some ads.
